pyi.py

```diff
@@ -1970,11 +1970,10 @@ class PyiTreeChecker:
     def run(self) -> Iterable[Error]:
         assert self.lines is not None
         assert self.tree is not None
-        path = Path(self.filename)
-        if path.suffix == ".pyi":
+        if Path(self.filename).suffix == ".pyi":
```

Comment:
No need to construct a `Path` object for something this simple:

```suggestion
        if self.filename.endswith(".pyi"):
```

Reply:
Ah, good point! That means we no longer need to import `pathlib`, as well :)